sonic-xcvrd/xcvrd/xcvrd.py

```diff
@@ -1484,7 +1475,7 @@ def join(self):
 
 class SfpStateUpdateTask(threading.Thread):
     RETRY_EEPROM_READING_INTERVAL = 60
-    def __init__(self, namespaces, port_mapping, main_thread_stop_event, sfp_error_event):
+    def __init__(self, namespaces, port_mapping, sfp_obj_dict, main_thread_stop_event, sfp_error_event, helper_logger):
```

Comment:
can you use SysLogger() for helper_logger

https://github.com/sonic-net/sonic-buildimage/blob/master/src/sonic-py-common/sonic_py_common/syslogger.py

Reply:
@prgeor Addressed this now